addons/website/views/snippets/snippets.xml

```diff
@@ -1067,6 +1067,10 @@
         </we-select>
     </div>
 
+    <xpath expr="//we-select[@id='s_popup_position_opt']" position="attributes">
+        <attribute name="data-dependencies" add="layout_popup_opt" separator=","/>
+    </xpath>
+
```

Comment:
s_popup_position_opt doesnt exist ... (anymore ?)
can you remind me what this is for ?

Reply:
it's my bad that I have removed reference id.
we did this because it's related to a cookie so we moved there. here is the link that will remind you
https://github.com/odoo/odoo/pull/63634#discussion_r556422714

Reply:
It doesnt work and I dont think we still need it